src/envo/envo-edit.owl

```diff
@@ -28119,6 +28142,155 @@ SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000032> <http://purl.obolibrar
 SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000032> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0000057> <http://purl.obolibrary.org/obo/ENVO_00000298>))
 SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000032> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0002234> <http://purl.obolibrary.org/obo/ENVO_01000293>))
 
+# Class: <http://purl.obolibrary.org/obo/ENVO_03000033> (marine sediment)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "https://en.wikipedia.org/wiki/Pelagic_sediment#_note-8") <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_03000033> "Sediment which accumulates within an ocean basin.")
```

Comment:
@pbuttigieg drawing upon the Dr. Bach's marine geology lectures on the I will refine the definition to: 

> Sediment soured from either terrestrial system, marine organisms, seawater and or cosmogeneous sources, which is transported through the marine water column, accumulating within a marine ocean basin or seafloor.